src/QA/QA.Web/Mappings/ReviewMapper.cs

```diff
@@ -211,6 +213,15 @@ public async Task<ReviewViewModel> Map(VacancyReview review)
             return vm;
         }
 
+        private string GetPageTitle(ReviewHistoriesViewModel historiesVm, Guid reviewId, ManualQaOutcome? reviewManualOutcome)
+        {
+            var timeFrame = historiesVm.Items.First().ReviewId == reviewId ? "Latest" : "Historical";
+            var outcome = reviewManualOutcome.GetValueOrDefault().ToString().ToLower();
```

Comment:
This is fine but maybe we could add this to EnumExtensions.GetDisplayName()

Reply:
Enum.ToString() does the job, the extension will not add any further value.